task-11/js/app.js

```diff
@@ -0,0 +1,23 @@
+let calculator = {
+    add: function (number1, number2) {
+        return number1 + number2;
+    },
+    subtract: function (number1, number2) {
+        return number1 - number2;
+    },
+    multiply: function (number1, number2) {
+        return number1 * number2;
+    },
+    divide: function (number1, number2) {
+        return number1 / number2;
+    },
+    modulo: function (number1, number2) {
+        return number1 % number2
+    }
+}
+
+console.log("Addition : ", calculator.add(10, 20));
```

Comment:
1. What if I enter a character or a string ? 
2. Can you add error handlings ? like say if I enter non number , it should say enter only numbers